### DISCUSSION TITLE
Typescript support for Postgres tables

### DISCUSSION THREAD

**Discussion by YannHulot** [Etheonor/supanextail#10] [Feedback]
Supabase lets you import type definitions in your project that are synched with your Postgres schema. 

This way you avoid runtime errors when querying fields that do not exist ona table. 

If typescript support is enabled, you can create a GitHub action that runs once a day and checks that the types in the repo are in sync with the DB.

**Comment by himeshp** [comment]
i believe, supabase is working on typescript support for client-side API.
